namespaces/Representor.md

```diff
@@ -0,0 +1,141 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Object Element)
+
+The Resource represents a Hypermedia Resource, with it's available relations, transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `content`: (object)
+    - `relations` - (array[Relation]) - The relations from this resource.
+    - `transitions` - (array[Transition]) - The available transitions from this resource.
+    - `resources` - (array[Resource]) - The embedded relations (resources) from this resource.
+    - `attributes` - (object) - Any properties of the resource.
+    - `relation` - (string) - An optional relation to this resource.
+
```

Comment:
Focusing on what is the real content of a resource vs. some metadata – what about this:

## Resource (Object Element)
- `element`: resource (string, fixed)
- `attributes` 
    - `transitions` - (array[Transition]) - The available transitions from this resource
    - `resources` - (array[Resource]) - The embedded relations – resources – from this resource
- `content` (MSON Element) - The content of the Element
    
    Either as Refract primitives (imagine a JSON here) or as MSON elements. 
    
### Example

```json
{
    "element": "resource",
    "attributes": {
        "transitions": [
            {
                "relation": "self",
                "uri": "/user"
            }
        ]
    },
    "content": {
        "name": "John",
        "address": "john@apple.com"
    }
}
```

The idea there is that if you take the content you will get the actual resource data (content). Either directly or as an MSON